flink-runtime/src/test/java/org/apache/flink/runtime/resourcemanager/slotmanager/TaskManagerValidateInSlotManagerTest.java

```diff
@@ -69,13 +70,15 @@
 	private CompletableFuture<InstanceID> releaseFuture;
 	private ResourceActions resourceManagerActions;
 	private ManuallyTriggeredScheduledExecutor mainThreadExecutor;
+	private final AtomicInteger allocateResourceCalls = new AtomicInteger(0);
```

Comment:
We should reset this to `0` in `setup()`.
Please be aware that there's no guarantee on the orders that the test cases are executed.